.eleventy.js

```diff
@@ -21,13 +22,35 @@ module.exports = function (eleventyConfig) {
     return content;
   });
 
+  eleventyConfig.addFilter("dateFormat", (dateString) => {
+    const date = new Date(dateString);
+    const day = date.getDate();
+    const monthNames = ["January", "February", "March", "April", "May", "June", "July", "August", "September", "October", "November", "December"];
+    const month = monthNames[date.getMonth()];
+    const year = date.getFullYear();
+
+    return `${day} ${month} ${year}`;
+  });
+
   eleventyConfig.addFilter("richTextToHTML", (value) => {
-    return documentToHtmlString(value);
+      return documentToHtmlString(value, {
+        renderNode: {
+          [BLOCKS.EMBEDDED_ASSET]: ({ data: { target: { fields }}}) =>
+          `
+            <figure>
+              <img src="${fields.file.url}" alt="${fields.description}" loading="lazy"/>
+              <figcaption>${fields.title}</figcaption>
+            </figure>
+          `,
+        }
+      });
   });
 
 
   // *** Rename regularly-changing assets, to prevent browser-cache issues ***
   (() => {
+
+    return;
```

Comment:
Do we want this?

Reply:
I'm going to revert that change - thank you